consensus/polybft/state_store_governance.go

```diff
@@ -36,17 +40,27 @@ type GovernanceStore struct {
 
 // initialize creates necessary buckets in DB if they don't already exist
 func (g *GovernanceStore) initialize(tx *bolt.Tx) error {
-	_, err := tx.CreateBucketIfNotExists(governanceEventsBucket)
-	if err != nil {
-		return fmt.Errorf("failed to create bucket=%s: %w", string(governanceEventsBucket), err)
+	if _, err := tx.CreateBucketIfNotExists(networkParamsEventsBucket); err != nil {
+		return fmt.Errorf("failed to create bucket=%s: %w",
+			string(networkParamsEventsBucket), err)
+	}
+
+	if _, err := tx.CreateBucketIfNotExists(forkParamsEventsBucket); err != nil {
+		return fmt.Errorf("failed to create bucket=%s: %w",
+			string(forkParamsEventsBucket), err)
 	}
 
-	if _, err = tx.CreateBucketIfNotExists(clientConfigBucket); err != nil {
+	if _, err := tx.CreateBucketIfNotExists(clientConfigBucket); err != nil {
 		return fmt.Errorf("failed to create bucket=%s: %w",
 			string(clientConfigBucket), err)
 	}
 
-	if _, err = tx.CreateBucketIfNotExists(lastProcessedGovernanceBlockBucket); err != nil {
+	if _, err := tx.CreateBucketIfNotExists(clientConfigBucket); err != nil {
```

Comment:
Should use lastProcessedGovernanceBlockBucket instead of clientConfigBucket?

Reply:
It was redundant: https://github.com/0xPolygon/polygon-edge/pull/1788/commits/80f1b9cbe34b29cd0aa674ba9e536e107f1a5a21